content/education/manage-coursework-with-github-classroom/get-started-with-github-classroom/about-github-classroom.md

```diff
@@ -0,0 +1,50 @@
+---
+title: About GitHub Classroom
+shortTitle: About GitHub Classroom 
+intro: '{% data variables.product.prodname_classroom %} is a space where teachers, organizations or school administrators can create assignments, collaborate with teaching assistants, and invite students in a single course.'
+product: 'Education'
+type: overview
+topics: 
+  - Fundamentals
+versions:
+  fpt: '*'
+redirect_from:
+  - /education/manage-coursework-with-github-classroom/about-github-classroom
+   
```

Comment:
```suggestion
```

We also don't have to include any redirects! Since this is a new article, there shouldn't be any users with an old URL/anyone expecting it to be in a different location 👍🏼